### ISSUE TITLE
Data in Figure3 is not matched between (a) and (b) .

### ISSUE THREAD

**Issue by Pcyslist** [aoiang/few-shot-NAS#4]
![image](https://user-images.githubusercontent.com/59328494/226360471-a7453a6b-65c3-4fa2-94eb-1cc54d81b57d.png)
Thank you for your great work. I have noticed that the data in Figure 3 (a) and (b) in the paper do not be matched. The predicted accuracy of the vertical axis one shot method in (a) does not even exceed 10%. Why can it be searched for more than 66% of one shot models in (b)? Similarly, the yellow points in (a) and the yellow lines in (b) do not match as well. Can you explain to me please? thank you.

**Comment by aoiang** [comment]
Thank you for your question. The vertical axis in figure(a) represents the predicted accuracy estimated by the one-shot supernet, while the vertical axis in figure(b) represents the true accuracy. In simpler terms, the vertical axis in figure(b) corresponds to the horizontal axis in figure(a). @Pcyslist

**Comment by Pcyslist** [comment]
Why does one shot NAS and few shot NAS have the concept of ground truth? Isn't ground truth a concept that only exists in network architectures trained from scratch? @aoiang

**Comment by aoiang** [comment]
> Why does one shot NAS and few shot NAS have the concept of ground truth? Isn't ground truth a concept that only exists in network architectures trained from scratch? @aoiang

Not exactly. One-shot and few-shot NAS are quick evaluation strategies. However, to measure the quality of the searched architectures, the ground truth is the only metric.

**Comment by Pcyslist** [comment]
How is the current best accuracy obtained on the vertical axis of figure (b)? Isn't the specific network architecture obtained through the RE sampling supernetwork (sub-supernetwork), and then the accuracy rate measured on the verification set by the supernetwork weight sharing to the specific network architecture?

**Comment by aoiang** [comment]
No, there is the ground truth accuracy. I trained them from scratch to get their true accuracy.

**Comment by Pcyslist** [comment]
Now that the vertical axis in figure(b) corresponds to the horizontal axis in figure(a). why the ground truth of different NAS method in figure(b)  is different ? shouldn't the ground truth be the same?

**Comment by aoiang** [comment]
They are not different, the vertical axis in figure(b) is the current best accuracy(ground truth), and the horizontal axis in figure(b) is the number of samples(architectures) searched by regularized evolution with one-shot, few-shot, and vanilla evaluation.

**Comment by Pcyslist** [comment]
Do you mean that once a current best architecture is searched by RE, you train it from scratch to obtain its true accuracy as the current best accuracy ?&nbsp;



---Original---
From: "Yiyang ***@***.***&gt;
Date: Mon, Mar 20, 2023 22:49 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [aoiang/few-shot-NAS] Data in Figure3 is not matched between (a)and (b) . (Issue #4)




 
They are not different, the vertical axis in figure(b) is the current best accuracy(ground truth), and the horizontal axis in figure(b) is the number of samples(architectures) searched by regularized evolution with one-shot, few-shot, and vanilla evaluation.

**Comment by aoiang** [comment]
I trained all the samples (from scratch) generated by different evaluation strategies, and the y-axis means the current best true accuracy of all generated samples.

**Comment by Pcyslist** [comment]
thanks very much! good luck to you!



---Original---
From: "Yiyang ***@***.***&gt;
Date: Mon, Mar 20, 2023 22:59 PM
To: ***@***.***&gt;;
Cc: ***@***.******@***.***&gt;;
Subject: Re: [aoiang/few-shot-NAS] Data in Figure3 is not matched between (a)and (b) . (Issue #4)




 
I trained all the samples (from scratch) generated by different evaluation strategies, and the y-axis means the current best true accuracy of all generated samples.
